Header header
uint8 aggregation_model 
float32 w1 
float32 w2 
float32 w3 
float32 w4 
float32 alpha 
float32 beta 